tf_cnn/pipeline-run/pipelinerun_4x-cpu.yml

```diff
@@ -20,7 +20,7 @@ spec:
   - name: device
     value: "cpu"
   - name: num_gpus
-    value: ""
+    value: "4"
```

Comment:
Is this change for using gpus ? maybe this needs to goto pipelinerun_4x-gpu.yml 
I also now think gpu resources value should be parameterized .I now see a large issue with benchmark task and I need to get  back to you if the benchmark task can itself be used for both cpu and gpu. IMO it is not possible but need to check with @raynalee4

Reply:
This change is for using CPUs, as per https://github.com/tensorflow/benchmarks/blob/master/scripts/tf_cnn_benchmarks/benchmark_cnn.py#L750,L752